.travis.yml

```diff
@@ -92,16 +99,15 @@ jobs:
     - stage: test
       <<: *job-template-osx
 
-    - stage: ui-lint
+    - stage: lint
```

Comment:
There is already a stage-lint job above. I don't know how come travis does not complain but doesn't this overshadow it?

Reply:
This is how the parallelization works from what I can see. It is like the two `test` stages you had above. This lint stage runs with a different job that includes node_js so the one is `python` and the other is `python` + `node` any for that reason it doesn't overshadow. If it was only python then it would.

Reply:
hmm okay let's see what the autopsy says

Reply:
We already had one though didn't we? the CI configuration did run for this PR :P